### PR TITLE
data_preprocess

### PR DESCRIPTION
**Pull request by wjn922**
fix some errors in download_video.sh
for activitynet, download.py relies on mmcv.
for gym, the virtual env name should be gym.

### CODE REVIEW

**Comment by innerlee** [comment]
Thanks for your contribution!

File: tools/data/activitynet/download_videos.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@
 conda env create -f environment.yml
 source activate activitynet
 pip install --upgrade youtube-dl
+pip install mmcv
```
Comment (innerlee) [inline tools/data/activitynet/download_videos.sh:7]: How about adding mmcv to env.yml

Comment (wjn922) [reply tools/data/activitynet/download_videos.sh:7]: That's also fine.

**Review by innerlee — APPROVED** [review]

File: tools/data/activitynet/download_videos.sh (hunk 1 above)
Comment (innerlee) [reply tools/data/activitynet/download_videos.sh:7]: Yeah its not important

**Pull request merged**